devito/yask/targets.py

```diff
@@ -40,7 +40,7 @@ def _pipeline(self, graph):
         avoid_denormals(graph)
         loop_wrapping(graph)
         if self.params['openmp']:
-            self.ompizer.make_openmp(graph)
+            self.ompizer.make_openmp_parallel(graph)
```

Comment:
did you mean `make_parallel` ? i can't find the definition of `make_openmp_parallel`